unpause/unpause.go

```diff
@@ -0,0 +1,129 @@
+package unpause
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-jose/go-jose/v4"
+	"github.com/go-jose/go-jose/v4/jwt"
+	"github.com/jmhodges/clock"
+)
+
+const (
+	// API
+	APIVersion = "v1"
+	APIPrefix  = "/sfe/" + APIVersion
+	GetForm    = APIPrefix + "/unpause"
+
+	// JWT
+	defaultIssuer   = "WFE"
+	defaultAudience = "SFE Unpause"
+)
+
+// JWTClaims represents the claims of a JWT token issued by the WFE for
+// redemption by the SFE. The JWT contains the following claims required for
+// unpausing:
+//   - Subject: the account ID of the Subscriber,
+//   - Version: the API version this JWT was created for,
+//   - Identifiers: a set of paused identifiers.
```

Comment:
It looks like we're flattening the ACMEIdentifier concept, by not including a type field to distinguish IP and DNS identifiers. I think that's a good call, since they have non-overlapping string representations. But let's document it.